app.rb

```diff
@@ -30,8 +31,7 @@ def set_locale(locale)
     else
       redirect '/'
     end
-
-    @board = @game.board.spaces.each_slice(3).to_a
+    @board = @game.current_state.each_slice(3).to_a
```

Comment:
The name `current_state` feels a little confusing here. It is giving me a board but the name of the function doesn't have board in it.